src/Components/Header.jsx

```diff
@@ -0,0 +1,14 @@
+const Header = () => {
+  return (
+    <div className="header">
+      <div>
```

Comment:
I would remove these divs. If you're using them for styling (e.g. separating components), it's better to handle this through CSS